pkgs/os-specific/linux/zfs/2_2.nix

```
@@ -18,16 +14,20 @@ callPackage ./generic.nix args {
   kernelCompatible = kernel: kernel.kernelOlder "6.13";
 
```

```suggestion
  kernelCompatible = kernel: kernel.kernelOlder "6.16";
```

https://github.com/openzfs/zfs/blob/zfs-2.2.8/META#L9